emsdk_manifest.json

```diff
@@ -741,6 +750,14 @@
     "os": "win",
     "custom_install_script": "emscripten_npm_install"
   },
+  {
+    "version": "releases-upstream-%releases-tag%",
+    "bitness": 64,
+    "uses": ["python-3.11.1-64bit"],
```

Comment:
Any idea why bitness must be specified in dependencies while arch does not need to?

Reply:
for some reason almost all targets defined this way and emsdk requires this for package names